### ISSUE TITLE
Issue when installing on Python 2.7.14

### ISSUE THREAD

**Issue by niloofaryousefi** [chathika/Py2NetLogo-Forked#2]
When using `pip` to install nl4py on Python 2.7.14, it keep trying to download nl4py version 0.1.1. It fails to recognize any of the later versions.

**Comment by niloofaryousefi** [comment]
`pip install nl4py`
```
Collecting nl4py
  Downloading https://files.pythonhosted.org/packages/4b/3f/3809e905008862b07d8284d4a4042c2677d0f08ca3be88b2e9a601aef756/NL4Py-0.1.1.tar.gz
NL4Py requires Python '>=3' but the running Python is 2.7.14
```

**Comment by chathika** [comment]
This bug has been addressed in [v0.2.0](https://github.com/chathika/NL4Py/releases/tag/v0.2.0)
